valhalla/baldr/edgetracker.h

```diff
@@ -15,7 +15,7 @@ namespace baldr {
 // giving something like 108mb of bits needed.
 struct bitset_t {
   typedef uint64_t value_type;
-  static const size_t bits_per_value = sizeof(value_type) * CHAR_BIT;
+  static const size_t bits_per_value = sizeof(value_type) * 8;
```

Comment:
Just not to bring `#include <limits>` as C++ anyway doesn't support platforms with other bit-per-byte count, check https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2024/p3477r0.html for more details